tsconfig.json

```
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "@/*": ["src/*"]
+      "@/*": ["src/*"],
+      "@/public/*": ["public/*"]
```

I suggest that we use `#` instead of `@/public`. Using `@/public` can lead to confusion about the directory location, as it may not be clear whether it is in the `src` directory.

```suggestion
      "#/*": ["public/*"]
```